### PR TITLE
Tests rating card

### PR DESCRIPTION
**Pull request by cristian0995**
Added tests for add-on rating cards.

### CODE REVIEW

File: tests/frontend/test_ratings.py [hunk 1]
```diff
@@ -502,3 +504,151 @@ def test_delete_developer_reply_to_review(selenium, base_url, variables):
...
+@pytest.mark.parametrize(
+    'addon_slug',
+    ('tabliss', 'facebook-container', 'bitwarden-password-manager', 'unloadtabs'),
+    ids=(
+        '5 stars rating add-on',
+        '4.3 stars rating add-on',
+        '4 stars rating add-on',
+        '3.5 stars rating add-on',
+    ),
+)
```
Comment (AlexandraMoga) [inline tests/frontend/test_ratings.py:567]: There is no guarantee that these addons will have these ratings and the slugs might also change, which will make the add-ons inaccessible. They are also hardcoded to addons present on stage. This test, in it's current format, is not durable. 

You should try a different method if you want to test the rating stars.

File: tests/frontend/test_ratings.py [hunk 2]
```diff
@@ -502,3 +504,151 @@ def test_delete_developer_reply_to_review(selenium, base_url, variables):
...
+
+@pytest.mark.nondestructive
+def test_meta_card_reviews_counter(selenium, base_url, variables):
+    # this test verifies that the total number of reviews is equal to the sum of one-star to five-star reviews
+    selenium.get(f'{base_url}/addon/{variables["addon_with_stats"]}')
+    page = Detail(selenium, base_url)
+    reviews_sum = 0
+    for el in page.stats.bar_rating_counts:
+        reviews_sum += int(el.text.split()[0])
+    assert int(page.stats.addon_reviews_stats.text.split()[0]) == reviews_sum
```
Comment (AlexandraMoga) [inline tests/frontend/test_ratings.py:654]: Are these two tests covering the stats card from the add-on detail page? If so, we already have tests for that section in the `test_addon_detail` suite. Check those tests to see if your scenarios are covered already. If not, add these tests to that suite.

**Review by AlexandraMoga — CHANGES_REQUESTED** [review]

File: tests/frontend/test_ratings.py (hunk 2 above)
Comment (cristian0995) [reply tests/frontend/test_ratings.py]: They are already covered, I deleted them

File: tests/frontend/test_ratings.py [hunk 3]
```diff
@@ -502,3 +504,151 @@ def test_delete_developer_reply_to_review(selenium, base_url, variables):
...
+        assert star.is_displayed()
+    assert rating_card.rating in range(6)
+    for i in range(5):
+        assert page.rating_card.number_of_reviews_with_specific_stars(i) >= 0
+    for rating_bar in rating_card.rating_bars:
+        assert rating_bar.is_displayed
+
+
+@pytest.mark.nondestructive
+def test_rating_card_title(selenium, base_url, variables):
```
[the remark this answers is not in the document]
Comment (cristian0995) [reply tests/frontend/test_ratings.py]: Ok

File: tests/frontend/test_ratings.py [hunk 4]
```diff
@@ -502,3 +504,151 @@ def test_delete_developer_reply_to_review(selenium, base_url, variables):
...
+def test_rating_card_loaded_correctly(selenium, base_url, variables):
+    selenium.get(variables["addon_version_page_url"])
+    page = Versions(selenium, base_url)
+    rating_card = page.rating_card
+    assert rating_card.addon_image.is_displayed()
+    assert rating_card.addon_title.is_displayed()
+    assert len(rating_card.addon_author_names) >= 1
+    for star in rating_card.rating_stars:
+        assert star.is_displayed()
+    assert rating_card.rating in range(6)
```
[the remark this answers is not in the document]
Comment (cristian0995) [reply tests/frontend/test_ratings.py:520]: Ok

File: tests/frontend/test_ratings.py (hunk 1 above)
Comment (cristian0995) [reply tests/frontend/test_ratings.py]: Changed it :)

**Review by AlexandraMoga — APPROVED** [review]

**Pull request merged**